roles/gluster-centos/defaults/main.yml

```diff
@@ -1,3 +1 @@
 ---
-# when true, tendrl-api is installed from a source repository
```

Comment:
My point here is, why remove file which might be needed later, so it's like place holder ... But I don't have strong opinion about this, so I have no problem with removing it.